### PR TITLE
Added a new rule package_tcp_wrappers_removed

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _A new rule as a part of CIS profile for SLE 12/SLE 15 _

#### Rationale:

- The rule supports CIS requirement 2.1.1 Ensure xinetd is not installed (Automated)

### CODE REVIEW

File: linux_os/guide/services/obsolete/inetd_and_xinetd/package_tcp_wrappers_removed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+prodtype: sle12,sle15
+
+title: 'Uninstall tcpd Package'
+
+description: |-
+    {{{ describe_package_remove(package="tcpd") }}}
+
+rationale: |-
+    Removing the <tt>xinetd</tt> package decreases the risk of the
```
Comment (Mab879) [inline linux_os/guide/services/obsolete/inetd_and_xinetd/package_tcp_wrappers_removed/rule.yml:11]:
```suggestion
    Removing the <tt>tcpd</tt> package decreases the risk of the
```

File: linux_os/guide/services/obsolete/inetd_and_xinetd/package_tcp_wrappers_removed/rule.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    xinetd service's accidental (or intentional) activation.
```
Comment (Mab879) [inline linux_os/guide/services/obsolete/inetd_and_xinetd/package_tcp_wrappers_removed/rule.yml:12]:
```suggestion
    tcpd service's accidental (or intentional) activation.
```

File: shared/references/cce-sle15-avail.txt [hunk 3]
```diff
@@ -5,7 +5,6 @@ CCE-92467-0
 CCE-92469-6
 CCE-92470-4
 CCE-92472-0
-CCE-92476-1
```
Comment (Mab879) [inline shared/references/cce-sle15-avail.txt:8]: Please check this, as there are merge conflicts.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts and respond to the instances of `xinetd` in the new rule.

**Comment by rumch-se** [comment]
Hello @Mab879 
I have done rebase. Because tcpd and xinetd are connected, I have tried to provide more details why we have to remove tcpd after the removal of xinetd. CIS covers only the removal of xinetd.
Have a nice day
Rumen

**Review by Mab879 — APPROVED** [review]
LGTM. Thanks for the PR!

**Comment by Mab879** [comment]
Automatus failures are expected due to the prodtype.

**Pull request merged**